Sources/Unbox.swift

```diff
@@ -526,16 +526,44 @@ public class Unboxer {
         return UnboxValueResolver<[String : V]>(self).resolveDictionaryValuesForKey(key, isKeyPath: isKeyPath, required: false, allowInvalidElements: false, valueTransform: { $0 })
     }
     
-    /// Unbox a required Dictionary containing collections
-    public func unbox<K: UnboxableKey, V: CollectionType>(key: String, isKeyPath: Bool = false) -> [K : V] {
+    /// Unbox a required Dictionary containing dictionaries
+    public func unbox<K: UnboxableKey, V: DictionaryLiteralConvertible>(key: String, isKeyPath: Bool = false) -> [K : V] {
```

Comment:
Make this constraint match the one on `UnboxValueResolver`; so `T: CollectionType, T: DictionaryLiteralConvertible, T.Generator == DictionaryGenerator<T.Key, T.Value>`. That way custom `DictionaryLiteralConvertible` objects are less likely to be matched.